pkg/minishift/network/settings.go

```diff
@@ -18,21 +18,34 @@ package network
 
 import (
 	"bytes"
+	"encoding/base64"
 	"fmt"
+	"strings"
 	"text/template"
 
+	"github.com/docker/machine/libmachine/drivers"
+	minishiftConfig "github.com/minishift/minishift/pkg/minishift/config"
 	"github.com/minishift/minishift/pkg/util/os/atexit"
 )
 
 const (
-	configureIPAddressMessage      = "-- Attempting to set network settings ..."
-	configureIPAddressFailure      = "FAIL\n   not supported on this platform or hypervisor"
-	configureNetworkScriptTemplate = `DEVICE={{.Device}}
+	configureIPAddressMessage                   = "-- Set the following network settings to VM ..."
+	configureRestartNeededMessage               = "Network settings get applied to the instance on restart"
+	configureIPAddressFailure                   = "FAIL\n   not supported on this platform or hypervisor"
```

Comment:
Extra space in front of  "not supported".

Reply:
@gbraad the messages look inconsistent e.g. `--` in front of `Set the following network settings to VM` and the `...` at `Set the following network settings to VM ...` . Extra space in front of `   not supported on this platform or hypervisor"`

Reply:
```
configureIPAddressMessage                   = "-- Set the following network settings to VM ..."
	configureRestartNeededMessage               = "Network settings get applied to the instance on restart"
```
The above ones are ok. But for the below ones the variable names are really long in some and I think we should drop the `configure` from the variable names. 
```
	configureIPAddressFailure                   = "FAIL\n   not supported on this platform or hypervisor"
	configureIPAddressAlreadySetFailure         = "Static IP address has already been assigned"
	configureNetworkNotSupportedMessage         = "The Minishift VM does not support network assignment"
	configureNetworkScriptStaticAddressTemplate = `DEVICE={{.Device}}
```

s/configureNetworkNotSupportedMessage/networkConfigNotsupported/
s/configureIPAddressFailure /ipconfigNotSupported/
s/configureNetworkScriptStaticAddressTemplate/staticAddressTemplateFotNetworkScript/
s/configureIPAddressAlreadySetFailure/ipAddressAlreadySetFailure/

Reply:
The messages "...", "FAIL\n   " will be fixed... they were from an earlier commit. So when squashed I can handle this anyways. it has bothered me for quite a while ;-)

Reply:
the more I look at the names, the rather I do not modify them now. they are all consistent and the `configure` prefix clearly indicates their usage. I do not think changing the names really conveys more information. Also, introducing an abbreviation, like `ipconfig` makes the intent more confusing; especially as `ipconfig` is an existing command for the same purposes of configuration.